optaplanner-examples/src/test/resources/org/optaplanner/examples/tsp/data/twoEnt.xml

```diff
@@ -0,0 +1,37 @@
+<TspSolution id="1">
+  <id>0</id>
+  <name>twoEntities</name>
+  <distanceType>AIR_DISTANCE</distanceType>
+  <distanceUnitOfMeasurement>distance</distanceUnitOfMeasurement>
+  <locationList id="2">
+    <TspAirLocation id="3">
+      <id>1</id>
+      <latitude>0.0</latitude>
+      <longitude>0.0</longitude>
+    </TspAirLocation>
+    <TspAirLocation id="4">
+      <id>2</id>
+      <latitude>1.0</latitude>
+      <longitude>0.0</longitude>
+    </TspAirLocation>
+    <TspAirLocation id="5">
+      <id>3</id>
+      <latitude>2.0</latitude>
+      <longitude>0.0</longitude>
+    </TspAirLocation>
+  </locationList>
+  <domicile id="6">
+    <id>1</id>
+    <location class="TspAirLocation" reference="3"/>
+  </domicile>
+  <visitList id="7">
+    <Visit id="8">
+      <id>2</id>
+      <location class="TspAirLocation" reference="4"/>
+    </Visit>
+    <Visit id="9">
+      <id>3</id>
+      <location class="TspAirLocation" reference="5"/>
+    </Visit>
+  </visitList>
+</TspSolution>
```

Comment:
About the files below, such as usa_ca_2716.xml: we 're not going put the xml version in the repository of those. The XML is very verbose and they take up too many space (in repo and in distro zips). Instead, load them from the import directory. Their TSP format files are a LOT smaller. We have other examples for which we do this trick. MachineReassingment IIRC.